src/schema/v2/me/artistRecommendations.ts

```diff
@@ -0,0 +1,79 @@
+import { GraphQLFieldConfig, GraphQLInt } from "graphql"
+import { connectionFromArray } from "graphql-relay"
+import { convertConnectionArgsToGravityArgs, extractNodes } from "lib/helpers"
+import { CursorPageable, pageable } from "relay-cursor-paging"
+import { createPageCursors } from "schema/v1/fields/pagination"
+import { ResolverContext } from "types/graphql"
+import gql from "lib/gql"
+import { artistConnection } from "schema/v2/artist"
+import { find } from "lodash"
+
+const MAX_ARTISTS = 50
```

Comment:
just for your info, we are only recommending twenty artists per user right now. however, we may change that so this looks good.

Reply:
Good catch and this is confusing indeed. It's an implementation detail and related to how we implement the pagination and not directly related to the amount of artists we query. I will add a comment to make it easier to understand.